src/config.ts

```diff
@@ -0,0 +1 @@
+export const API_URL = process.env.API_KEY ?? 'http://localhost:8888';
```

Comment:
This probably shouldn't be localhost for deployment

Reply:
it takes the env var first and goes to localhost if it doesnt exist

Reply:
Oh right, cool 👍